shell/common/animator.cc

```diff
@@ -175,9 +175,20 @@ void Animator::Render(std::unique_ptr<flutter::LayerTree> layer_tree) {
   frame_timings_recorder_->RecordBuildEnd(fml::TimePoint::Now());
 
   // Commit the pending continuation.
-  bool result = producer_continuation_.Complete(std::move(layer_tree));
-  if (!result) {
+  PipelineProduceResult result =
+      producer_continuation_.Complete(std::move(layer_tree));
+
+  if (!result.success) {
     FML_DLOG(INFO) << "No pending continuation to commit";
+    return;
+  }
+
+  if (!result.is_first_item) {
+    // It has been successfully pushed to the pipeline but not as the first
+    // item. Eventually the 'Rasterizer' will consume it, so we don't need to
+    // notify the delegate.
+    FML_DLOG(INFO) << "Not the first item in the pipeline, ignore it";
```

Comment:
nit: this log is likely to be noisy. I know it's only in local unopt builds, but I'm not sure what action I'd ever want to take if I was debugging a local engine build. We should drop it.

Reply:
Done